### DISCUSSION TITLE
Allow for app restarts via the dashboard

### DISCUSSION THREAD

**Discussion by adamtrip** [microsoft/aspire#5598] [Ideas]
It would be nice to be able to restart individual apps via the dashboard. I don't know how achievable is such thing while in production due to the different possibilities of deployments (docker, k8s, etc).

However, while in development mode, running the Host via IDE or terminal, would be quite useful to be able to restart apps on a button click. It's quite handy, for instance, on scenarios with Node apps where sometimes we just need to restart them for some reason and having to restart the whole solution can take some time.

**Comment by davidfowl** [comment]
This is planned for aspire 9 https://github.com/dotnet/aspire/issues/295

**Reply by adamtrip** [reply]
Awesome! Looking forward to it!

**Reply by davidfowl** [reply]
https://learn.microsoft.com/en-us/dotnet/aspire/whats-new/dotnet-aspire-9-release-candidate-1
